### DISCUSSION TITLE
RFE: port pkcs7 support to openssl

### DISCUSSION THREAD

**Discussion by kloczek** [hughsie/libjcat#69] [Ideas]
Most of the OSS projects are using openssl instedad gnutls.

It would be good to have possibility to build libjcat against openssl.

**Comment by hughsie** [comment]
The OpenSSL API is simply terrible for this kind of thing. I'd be happy if someone wanted to write a backend that used openssl, but it's not something I want to do myself, sorry.

**Comment by kloczek** [comment]
Jus please keep that issue ticket opened as the confirmation that you will accept patch with such change :)

Maybe someone will do that :P
